README.md

```diff
@@ -1 +1,51 @@
-# cloud-demo
\ No newline at end of file
+# Arkime Cloud Demo
+
+The goals of this project are 1) provide a demonstration of how Arkime can be deployed in a cloud-native manner and 2) provide scripting to enable users to easily begin capturing the traffic in their existing AWS cloud infrastructure.
+
+The AWS Cloud Development Kit (CDK) is used to perform infrastructure specification, setup, management, and teardown.  You can learn more about infrastructure-as-code using the CDK [here](https://docs.aws.amazon.com/cdk/v2/guide/home.html).
+
+## How to run the demo
+
+### Pre-requisites
+
+* REQUIRED: A copy of the repo on your local host
+* REQUIRED: A properly installed/configured copy of Node ([see instructions here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
```

Comment:
So, might be nice to just use the arkime node install and packages. Depends if we are looking at this as standalone or not.

Reply:
I'm but a novice in the ways of Node and NPM; can you expand on that first statement a bit?

Regarding standalone - depends on how we define that.  My inclination is to pull the Arkime installables from their public distro location rather than deploying from a local copy at least for the first pass of this.  In that sense, I guess my thought is that this is "standalone".  What did you have in mind?

Reply:
I think for now what you have is fine. Eventually we might want to use the node that is installed in /opt/arkime/bin just so its known what folks are running and its already installed.